cfn-resources/util/util.go

```diff
@@ -110,7 +110,7 @@ func NewMongoDBClient(req handler.Request, profile *string) (*mongodbatlas.Clien
 	return client, nil
 }
 
-func getAPIKeys(req handler.Request, profile string) (*DeploymentSecret, *handler.ProgressEvent) {
+func GetAPIKeys(req handler.Request, profile string) (*DeploymentSecret, *handler.ProgressEvent) {
```

Comment:
```suggestion
func getAPIKeys(req handler.Request, profile string) (*DeploymentSecret, *handler.ProgressEvent) {
```

this should not be public

Reply:
Reverted